docs/documentation/platform/gateways/relay-deployment.mdx

```diff
@@ -0,0 +1,424 @@
+
+- **Organization Relays** - Deploy and manage your own dedicated relay servers. Choose this for lower latency, custom network policies, compliance requirements, or when you need full control over the relay infrastructure.
+
+<Info>
+  **Note:** If you're using Infisical Cloud, you likely don't need to deploy
+  relays at all - you can use the managed US/EU relays. This guide is for users
+  who need to deploy their own relay infrastructure.
+</Info>
+
+<AccordionGroup>
+  <Accordion title="Instance Relays">
+    Instance relays require **two-step configuration** with matching secrets:
+
+    **Step 1: Instance Admin Configuration**
+
+    The Infisical instance administrator must first set the relay auth secret in the instance environment:
+
+    ```bash
+    # In the Infisical instance environment variables
+    RELAY_AUTH_SECRET=<shared-secret-value>
+    ```
+
+    **Step 2: Relay Deployment**
+
+    Deploy the relay using the **same secret value**:
+
+    ```bash
+    # Set the matching secret for relay deployment
+    export INFISICAL_RELAY_AUTH_SECRET=<shared-secret-value>
+
+    # Start the instance relay
+    infisical relay start --type=instance --host=<public-ip> --name=<relay-name>
+    ```
+
+    **Install as daemon (Linux only):**
+
+    ```bash
+    # Instance relay
+    sudo infisical relay systemd install --type=instance --name=<name> --host=<host> --relay-auth-secret=<secret>
+
+    # Start the service
+    sudo systemctl start infisical-relay
+
+    # Uninstall
+    sudo infisical relay systemd uninstall
+    ```
+
+    <Warning>
+      The `RELAY_AUTH_SECRET` (instance environment) and
+      `INFISICAL_RELAY_AUTH_SECRET` (relay environment) **must match exactly** for
+      authentication to work.
+    </Warning>
+
+  </Accordion>
+
+  <Accordion title="Organization Relays">
+    Deploy your own relay server for full control. Organization relays support all authentication methods:
+
+    ```bash
+    # Token auth
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --token=<token>
+
+    # Universal auth
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --auth-method=universal-auth --client-id=<id> --client-secret=<secret>
+
+    # Other auth methods (kubernetes, aws-iam, gcp-id-token, azure, etc.)
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --auth-method=<method> --machine-identity-id=<id>
+    ```
+
+    **Install as daemon (Linux only):**
+
+    ```bash
+    # Organization relay
+    sudo infisical relay systemd install --token=<token> --name=<name> --host=<host> --type=org
+
+    # Start the service
+    sudo systemctl start infisical-relay
+
+    # Uninstall
+    sudo infisical relay systemd uninstall
+    ```
+
+  </Accordion>
+</AccordionGroup>
+
+## Network Configuration
```

Comment:
Most of these should just become simple steps. See how gateway deployment as example